orders/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.db.models.fields import IntegerField
+
+from core.models     import Timestamp
+from users.models    import User, Address
+from products.models import Product
+# Create your models here.
+class Order(Timestamp):
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    address      = models.ForeignKey(Address, on_delete=models.CASCADE)
+    order_status = models.ForeignKey("Order_status", on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'orders'
+
+class Order_item(Timestamp):
+    order        = models.ForeignKey("Order", on_delete=models.CASCADE)
+    product      = models.ForeignKey(Product, on_delete=models.CASCADE)
+    quantity     = models.IntegerField(default=0)
+    items_status = models.CharField(max_length=30)
+
+    class Meta:
+        db_table = 'order_items'
```

Comment:
class명은 camelCase로 만들어주세요~

```suggestion
class OrderItem(Timestamp):
    order        = models.ForeignKey("Order", on_delete=models.CASCADE)
    product      = models.ForeignKey(Product, on_delete=models.CASCADE)
    quantity     = models.IntegerField(default=0)
    status        = models.CharField(max_length=30)

    class Meta:
        db_table = 'order_items'
```